dev/bots/test/analyze-test-input/root/packages/foo/serviceaccount.enc

```diff
@@ -0,0 +1 @@
+�
```

Comment:
Intended?  Will this not break the decryption?

Reply:
This is just a test file.